### PR TITLE
Billing UX updates

### PR DESCRIPTION
**Pull request by ackernaut**
Addresses parts of #1980 WIP

BL = Billing Leader

### Even Newer Checklist
- [x] Adds “Upgrade to Professional” CTA to the top-left user menu. It shows when the user is a BL and is a member of 1 or more free orgs. If a member of 1 free org, the CTA routes to the billing squeeze for that org. If a member of 2 or more free orgs, the CTA routes to the organization list.
- [x] Adds a CTA “Upgrade to Professional” for free orgs in the organization list. Tapping the CTA routes the user to the billing squeeze.
- [x] Updates the styles of the billing squeeze for billing leaders
- [x] Makes a bunch of components used for billing, panels, settings, etc. more consistent visually, and now using `react-emotion`

### Newer Checklist

Scoping upgrade CTAs to the Billing Leaders only as a first step that can get shipped sooner. We need to write another issue to implement the ability for non-BL org members to view org information like members including BLs or tier (but no CC and invoicing!).

- [ ] Adds “Upgrade to Professional” CTA to the top-left user menu. It shows when the BL is a member of 1 or more free orgs. If a member of 1 free org, the CTA routes to the billing squeeze for that org. If a member of 2 or more free orgs, the CTA routes to the organization list.
- [ ] Adds a CTA “Upgrade to Professional” for free orgs in the organization list. Tapping the CTA routes the user to the billing squeeze.
- [ ] Updates the styles of the billing squeeze for billing leaders
- [ ] For Billing Leaders only: adds a link with org name to the team dash header. Adds an upgrade squeeze link when the team is on a personal org.
- [ ] Adds an upgrade CTA to the Team Settings panel visible to the BL if the team is free.
- [ ] Makes a bunch of components used for billing, panels, settings, etc. more consistent visually, and now using `react-emotion`

### Older Checklist

Update these based on what’s changed in the future and add to a part ii issue:

- [ ] Adds “Upgrade to Professional” CTA to the top-left user menu. It shows when the user is a member of 1 or more free orgs. If a member of 1 free org, the CTA routes to the billing squeeze for that org. If a member of 2 or more free orgs, the CTA routes to the organization list.
- [ ] Adds a CTA “Upgrade to Professional” for free orgs in the organization list. Tapping the CTA routes the user to the billing squeeze.
- [ ] Updates the styles of the billing squeeze for billing leaders
- [ ] Adds the ability for org members to nudge billing leaders in the org squeeze view. There is conditional presentation for the org if it has a single BL (primary button to nudge) or multiple BLs (rows with inline controls to nudge per BL).
- [ ] Non billing leaders can see a read-only list of org members. The user can leave the organization inline on their own row. The user can clearly see which users are billing leaders and which are inactive. The non-BLs can see this view when on the personal or professional tiers.
- [ ] Adds a link with org name to the team dash header. Adds an upgrade squeeze link when the team is on a personal org.
- [ ] The non-BL org member does not have Billing and Members navigation when viewing a Professional org. They only see the org members view in read-only mode, where they can leave the organization on their own user row.
- [ ] There is a upgrade nudge notification that can be sent to the viewer. “Elizabeth wants to upgrade Parabol, Inc. to the Professional tier.” (See Options [routes to billing squeeze] or ✔️ [clears] ). I think that 1 user can only nudge 1 billing leader once. A billing leader can receive the notification from every org member that nudges them.
- [ ] Makes a bunch of components used for billing, panels, settings, etc. more consistent visually, and now using `react-emotion`

[See Figma](https://www.figma.com/file/DXqOZNFNbfvkBBxvPCKgfiNW/Nudges-to-Upgrade)

### CODE REVIEW

**Comment by ackernaut** [comment]
Okie dokie @mattkrick this is ready for you, see the New Checklist. I’ll make a new issue to capture enhancements for allowing any org member to see org stuff. I have an idea for a minimal first step, too.

**Comment by ackernaut** [comment]
Update here: I’m going to give 1 more quick pass and leave @mattkrick  with a minimal list of things to get this merged. There are some considerations that can be shared with #1967 on the team settings view and #2059 which @mattkrick has behind this. Once again, this partially addresses #1980 on the Sprint board.

**Comment by ackernaut** [comment]
Ok @mattkrick if you could take care of checklist item 1 (adds conditional upgrade CTA to user menu), then we can merge this. Moving back to “To Do” on the sprint board.

File: src/universal/modules/integrations/components/ServiceRow/ServiceRow.js [hunk 1]
```diff
@@ -1,104 +1,104 @@
...
-    fontSize: appTheme.typography.s4,
-    fontWeight: 600,
-    flex: 1,
-    lineHeight: '1.625rem',
-    verticalAlign: 'middle'
-  }
-});
-
-export default withStyles(styleThunk)(ServiceRow);
+// import PropTypes from 'prop-types';
```
Comment (mattkrick) [inline src/universal/modules/integrations/components/ServiceRow/ServiceRow.js]: can remove?

File: src/universal/modules/notifications/components/NudgeToUpgrade/NudgeToUpgrade.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
+// import {css} from 'react-emotion';
```
Comment (mattkrick) [inline src/universal/modules/notifications/components/NudgeToUpgrade/NudgeToUpgrade.js]: remove?

File: src/universal/modules/teamDashboard/components/IntegrationRow/IntegrationRow.js [hunk 3]
```diff
@@ -1,37 +1,6 @@
...
-import ui from 'universal/styles/ui';
+import styled from 'react-emotion';
+import {Row} from 'universal/components';
 
-const IntegrationRow = (props) => {
-  const {
-    children,
-    styles
-  } = props;
+const IntegrationRow = styled(Row)({paddingLeft: 0});
```
Comment (mattkrick) [inline src/universal/modules/teamDashboard/components/IntegrationRow/IntegrationRow.js]: YESSSSS so clean!

File: src/universal/modules/userDashboard/components/OrgPlanSqueeze/OrgPlanSqueeze.js [hunk 4]
```diff
@@ -22,127 +133,155 @@ class OrgPlanSqueeze extends Component {
...
-            <p className={css(styles.copy)}>
-              {'Your current plan.'}<br />
-              {'The basics, for free!'}
-            </p>
+
+    const isBillingLeader = true;
+
+    // NOTE: hasUpgradeNudge is true when the billing leader has received a notification from the current user for this org
+    //       the billing leader can have 1 notification from every org member that nudges
+    const billingLeaders = [
```
Comment (mattkrick) [inline src/universal/modules/userDashboard/components/OrgPlanSqueeze/OrgPlanSqueeze.js]: remove?

File: src/universal/modules/userDashboard/components/OrgPlanSqueeze/OrgPlanSqueeze.js [hunk 5]
```diff
@@ -22,127 +133,155 @@ class OrgPlanSqueeze extends Component {
...
+            <TierPanelBody>
+              <CopyWithStatus>
+                <b>{'Your current plan.'}</b><br />
+                {'The basics, for free!'}
+              </CopyWithStatus>
+            </TierPanelBody>
+          </TierPanel>
+          {/* Professional Panel */}
+          <TierPanel tier={PRO}>
+            <TierPanelHeader tier={PRO}>{PRO_LABEL}</TierPanelHeader>
```
Comment (mattkrick) [inline src/universal/modules/userDashboard/components/OrgPlanSqueeze/OrgPlanSqueeze.js]: remove?

**Comment by mattkrick** [comment]
ok @ackernaut functionality added & PR reviewed. nice work! just a few things we should remove before we push to master

**Comment by mattkrick** [comment]
merging, will move the new checklist to a new PR

**Pull request merged**